docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -161,55 +161,27 @@ Use webhooks and the below snippet to integrate with GitHub.
 
 ```Groovy
 pipeline {
-  agent {
-    kubernetes {
-      yaml """
-        apiVersion: v1
-        kind: Pod
-        spec:
-          containers:
-          - name: semgrep
-            image: 'returntocorp/semgrep-agent:v1'
-            command:
-            - cat
-            tty: true
-        """
-      defaultContainer 'semgrep'
-    }
-  }
-
-environment {
-    SEMGREP_RULES = "p/security-audit p/secrets" // more at semgrep.dev/explore
-    SEMGREP_BASELINE_REF = "origin/${env.CHANGE_TARGET}"
-
-    // == Optional settings in the `environment {}` block
+  agent any
+    environment {
+      SEMGREP_RULES = "p/security-audit p/secrets" // more at semgrep.dev/explore
```

Comment:
can we change to auto?

Reply:
Sure thing; as in, removing the "SEMGREP_RULES" variable and adding "--config auto . " to the command?

Reply:
The latter would be great @holdenmcg

Reply:
Commented out the 'environment' block, setting it to --config auto by default.